fixtures/dom/src/components/fixtures/hydration/index.js

```diff
@@ -0,0 +1,69 @@
+import './hydration.css';
+import {SAMPLE_CODE} from './data';
+import {LiveProvider, LiveEditor, LiveError} from 'react-live';
+import * as buble from 'buble';
```

Comment:
Buble is the best!

Reply:
Actually... spoke too soon (buble is still great) it turns out I needed Babel. Buble uses some `__proto__` tricks that break in IE9/10. 

To get IE9 support, I ended up pulling in babel/standalone with a few polyfills.